src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -37,6 +37,58 @@
  * Interface (SPI) for locating HTTP server implementations based on the
  * {@code com.sun.net.httpserver} API.
+ * <p>
+ * <b id="httpserverprops">System properties used by the HTTP server API</b>
+ * <p>
+ * The following is a list of JDK specific system properties used by the HTTP
+ * server implementation.
+ * <ul>
+ * <li><p><b>{@systemProperty sun.net.httpserver.idleInterval}</b> (default: 30 sec)<br>
+ * Maximum duration in seconds which an idle connection is kept open. This timer
+ * has an implementation specific granularity that may mean that idle connections are
+ * closed later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpserver.maxConnections}</b> (default: -1)<br>
+ * The maximum number of open connections at a time. This includes active and idle connections.
+ * If zero or negative, then no limit is enforced.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxIdleConnections}</b> (default: 200)<br>
+ * The maximum number of idle connections at a time.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.drainAmount}</b> (default: 64K)<br>
+ * The maximum number of bytes that will be automatically read and discarded from a
+ * request body that has not been completely consumed by its
+ * {@link com.sun.net.httpserver.HttpHandler HttpHandler}. If the number of remaining
+ * unread bytes are less than this limit then the connection will be put in the idle connection
+ * cache. If not, then it will be closed.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqHeaders}</b> (default: 200)<br>
+ * The maxiumum number of header fields accepted in a request. If this limit is exceeded
+ * while the headers are being read, then the connection is terminated and the request ignored.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a request. In practice, this is a function of
+ * request size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}. This timer has an implementation specific granularity
+ * that may mean requests are aborted later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxRspTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a response. In practice, this is a function of
+ * response size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}. This timer has an implementation specific granularity
+ * that may mean responses are aborted later than the specified interval.
```

Comment:
I think the `maxReqTime` and the `maxRspTime` will need a bit more clarification on how they differ. Specifically the `maxReqTime` as used by the current server implementation is the duration to read the entire request content. It however, doesn't include the time it takes to generate the response nor process the request through the handler.

Reply:
Okay, I can specifically refer tp "request headers and body" and "response headers and body".

So, for example replace "The maximum duration in milliseconds of a request." with

"The maximum time in milliseconds allowed to receive the request headers and body".